validator/src/bootstrap.rs

```diff
@@ -1161,42 +1199,50 @@ mod with_incremental_snapshots {
         trusted_validators
             .iter()
             .for_each(|trusted_validator| {
+                // First get the Crds::SnapshotHashes for each trusted validator and add them as
+                // the keys in the trusted snapshot hashes HashMap.
+                let mut full_snapshot_hashes = Vec::new();
+                cluster_info.get_snapshot_hash_for_node(trusted_validator, |snapshot_hashes| {
```

Comment:
As a follow-up low-priority clean-up PR, could you please scrub `validator/...` of references to "trusted" validators (like `trusted_validator` here and in the comment above this line).  We call these "known" validators now at the command-line and in the docs to avoid using the loaded word "trust"

Reply:
Will do!

Reply:
Follow-up issue for this task: #21169